Header header
int32 drone_id
int32 target_id
string signal
